api/v1/manager_types.go

```diff
@@ -22,38 +22,12 @@ import (
 
 // ManagerSpec defines configuration for the Calico Enterprise manager GUI.
 type ManagerSpec struct {
-	// Auth defines the authentication strategy for the Calico Enterprise manager GUI.
-	// +optional
-	Auth *Auth `json:"auth,omitempty"`
```

Comment:
Same here - this might break existing users.

We'll need to spell this out really loudly in the release notes.

Reply:
I added deprecation. Thanks for bringing this up.